### ISSUE TITLE
Allow stdout for passing tests

### ISSUE THREAD

**Issue by evilrobot-01** [nextest-rs/nextest#757]
Hey there!

I have a rare scenario where I would like to capture the logging output for a handful of successful tests, which ultimately end up being converted to JUnit Xml format. I tried `--success-output final`, but it has no effect for passing tests, presumably due to https://github.com/nextest-rs/nextest/blob/3a7a63739c99fb7757a2b382ea5c6b72d9599ef9/nextest-runner/src/reporter/aggregator.rs#L165

Could you possibly confirm whether this is indeed still on the TODO list and I am not missing something please? 🙂

**Comment by sunshowers** [comment]
Thanks for the feature request! This would be a great feature to add for sure. We could expose it as an option under the JUnit configuration.

**Comment by sunshowers** [comment]
Going to release support for new config options:

```toml
[profile.default.junit]
store-success-output = true  # false by default
```

and also as a [per-test override](https://nexte.st/book/per-test-overrides.html):

```toml
[[profile.default.overrides]]
filter = 'test(my-tests)'
junit.store-success-output = true
```

so you can choose to do this for a subset of tests.

**Comment by sunshowers** [comment]
Planning to release this sometime this weekend.

**Comment by sunshowers** [comment]
whoops, this was released in cargo-nextest 0.9.49.
